src/Atmos/Model/declare_prognostic_vars.jl

```diff
@@ -1,21 +1,21 @@
 ##### Prognostic variable
 
 export Mass, Momentum, Energy, ρθ_liq_ice
-export Moisture, TotalMoisture, LiquidMoisture, IceMoisture
-export Precipitation, Rain, Snow
+export AbstractMoistureVariable, TotalMoisture, LiquidMoisture, IceMoisture
+export AbstractPrecipitationVariable, Rain, Snow
```

Comment:
I agree that adding `Prognostic` suffix would nicely mirror `TotalEnergyVariable` and `TotalEnergyModel`, but I think we might get hit much more by the formatter with changing the concrete types because they're more widely used-- I'm inclined to try these changes separately.

Reply:
Also, would be nice to get thoughts from others on this!

Reply:
Also also, some variable names are already very long and difficult to name (e.g., `en_ρaθ_liq_q_tot_cv` for EDMF). For the EDMF team this definitely beats the verbose `EnvironmentalLiquidIcePotentialTemperatureAndTotalMoistureCovariance` 😉.